content/pages/english.mdx

```diff
@@ -12,7 +12,7 @@ menu: 6
 
 Queer Hangout is not only a friendgroup, but also a group of activists that work against loneliness in the queer community. Many feel they don't belong in the queer community, maybe because they're in a wheelchair and the venue has no ramps, or because you belong to a exposed minority like muslims or sex workers. We try to make everyone feel welcome, and work with every member and relevant organizations (Like [Skeiv Verden](https://skeivverden.no)) to achieve this.  
```

Comment:
Lenker til eksterne nettsider må skrives i html, fordi man må legge inn en `rel="nofollow"` attribute for at Google boter skal skjønne at siden vi lenker til ikke er vår. Du kan også legge på en `target="_blank"` for at siden skal åpnes i en ny fane. Du kan bytte lenken til Skeiv verden med denne:

`<a href="https://skeivverden.no" rel="nofollow" target="_blank">Skeiv Verden</a>`